crypto/context.c

```diff
@@ -81,6 +82,9 @@ static int context_init(OSSL_LIB_CTX *ctx)
 {
     int exdata_done = 0;
 
+    if (!CRYPTO_THREAD_init_local(&ctx->rcu_local_key, NULL))
+        return 0;
+
     ctx->lock = CRYPTO_THREAD_lock_new();
     if (ctx->lock == NULL)
         return 0;
```

Comment:
hmm, I suppose it does, though it wouldn't be an actual memory leak (at least not in the pthreads case), since a pthread_key_t is just an integer.  It would leak the key index in glibc.  Either way, I'll fix it up.

Reply:
fixed